cli/tests/integrations/test_service.py

```diff
@@ -16,29 +20,52 @@ def zk_znode(request):
     return request
 
 
+def setup_module(module):
+    package_install('chronos', True)
+
+
+def teardown_module(module):
```

Comment:
Should we standardize on using these methods instead of `pytest.fixture(scope=module)`? I am okay with either way. There are advantages and disadvantages to either one. I just want us to pick one. cc @tamarrow